src/main/extras/app/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -x
+set -e
+
+PWFILE="/tmp/jmxremote.password"
+function createJmxPassword() {
+    PASS="$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-32})"
```

Comment:
I don't see this being called anywhere with an argument. We could probably simplify this to `head -c32`, unless you want to keep this length parameter for the future.

Reply:
Yea it is currently unused. It can always be put back in the future, so I'll take it out for now and leave it at fixed-length of 32.